### ISSUE TITLE
[Admin] Problem with customer account credentials

### ISSUE THREAD

**Issue by 4oXdev** [Sylius/Sylius#7953]
Problem with enabling account credentials for deleted customers.

**Comment by 4oXdev** [comment]
If customer has been deleted from admin panel you can't re-enable credentials for it correctly. It's looks like enabled and customer can logged, but in admin panel customer account credentials is switched off and if you wish delete this user you got error 'Invalid csrf token'.

**Comment by jdeveloper** [comment]
The invalid csfr token occurs if user id and customer id aren't the same.

**Comment by jdeveloper** [comment]
Hie @pjedrzejewski, 

I'm trying to fix the problem of deleting the user, I mean, I'm trying actualy to send a PR.Well I found out the problem has to do with checking the csfr token. Since the ids are not equal, when I access the customer show page, the csfr id stored in session is the one of the customer, so when the broweser redirects to the URL for deleting the user, it does not exist in session a csfr token for this id.

Any suggestion where the token is replaced so I can look if theres a way to change it?

**Comment by jdeveloper** [comment]
Well, I just discoverd that you can configure to disable csrf protection for a request. I override it for now in our project.

```
sylius_admin_shop_user_delete:
    path: /{id}
    methods: [DELETE]
    defaults:
        _controller: sylius.controller.shop_user:deleteAction
        _sylius:
            section: admin
            redirect: referer
            permission: true
            csrf_protection: false
```

**Comment by jdeveloper** [comment]
But I all ready thought a possible solution even now I don't have time to implent it. It would be to configure optional in the route the expression to get the id to check the csrf token like:

```
sylius_admin_shop_user_delete:
    path: /{id}
    methods: [DELETE]
    defaults:
        _controller: sylius.controller.shop_user:deleteAction
        _sylius:
            section: admin
            redirect: referer
            permission: true
            csrf_id_param: expr($object.getCustomer()->getId())
```

**Comment by pamil** [comment]
From what I know of, it has been fixed a few months ago, the most recent 1.1.x, 1.2.x, 1.3.x releases should not have this issue.
